data/json/itemgroups/locations_commercial.json

```diff
@@ -1860,5 +1860,32 @@
       [ "recipe_fauxfur", 5 ],
       [ "textbook_tailor", 3 ]
     ]
+  },
+  {
+    "id": "casings",
```

Comment:
This definitely is too general to belong in locations_commercial, even if it was originally for a shooting range

Reply:
I've now moved casings to ammo.json instead